analyses/cnv-chrom-plot/cn_status_heatmap.Rmd

```diff
@@ -396,12 +396,17 @@ histologies <-
 Make a color key that's formatted for ComplexHeatmap. 
 
 ```{r}
+# Get a distinct version of the color keys
+histologies_color_key_df <- histologies %>%
+  dplyr::select(display_group, hex_codes) %>%
+  dplyr::distinct()
+
 # Make color key specific to these samples
-histologies_color_key_filtered <- unique(histologies$hex_codes)
-names(histologies_color_key_filtered) <- unique(histologies$display_group)
+histologies_color_key <- unique(histologies_color_key_df$hex_codes)
```

Comment:
Do we need the `unique()` here still? The idea of using `distinct()` above was to remove this step. If  `unique(histologies_color_key_df$hex_codes) != histologies_color_key_df$hex_codes` we would presumably have a problem...

Reply:
Oh right! Yes. We should be able to delete that. Will delete.